finish/src/main/java/io/openliberty/guides/inventory/InventoryResource.java

```diff
@@ -81,8 +81,10 @@ public Response getPropertiesForHost(
         Properties props = manager.get(hostname);
         if (props == null) {
             return Response.status(Response.Status.NOT_FOUND)
-                            .entity("ERROR: Unknown hostname or the system service may "
-                                + "not be running on " + hostname)
+                            .entity("{ \"error\" : " 
+                                    + "\"Unknown hostname " + hostname 
+                                    + "or the resource may not be "
```

Comment:
Need a space before the "or" word as in the start directory

Reply:
Adjustment made